hack/feature-deploy.sh

```diff
@@ -2,11 +2,66 @@
 
 set -e
 
+if [ "$FEATURES_ENVIRONMENT" == "" ]; then
+	echo "[ERROR]: No FEATURES_ENVIRONMENT provided"
+	exit 1
+fi
+
+if [ "$FEATURES" == "" ]; then
+	echo "[ERROR]: No FEATURES provided"
+	exit 1
+fi
+
 # expect oc to be in PATH by default
-OC_TOOL="${OC_TOOL:-oc}"
+export OC_TOOL="${OC_TOOL:-oc}"
+
+# expect kustomize to be in PATH by default
+KUSTOMIZE="${KUSTOMIZE:-kustomize}"
+
+# Label 1 worker node
+echo "[INFO]:labeling 1 worker node with worker-rt"
+node=$(${OC_TOOL} get nodes --selector='node-role.kubernetes.io/worker' -o name | head -1)
```

Comment:
Small question. If my master nodes also have "worker" label - the first master will be chosen, is it ok?

[root@dell-r730-012 dev-scripts]# oc get nodes --selector='node-role.kubernetes.io/worker' -o name | head -1
node/master-0

[root@dell-r730-012 dev-scripts]# oc get node
NAME       STATUS   ROLES 
master-0   Ready    master,worker
master-1   Ready    master,worker
master-2   Ready    master,worker
worker-0   Ready    worker
worker-1   Ready    worker